conf/application.conf

```diff
@@ -75,7 +75,7 @@ application.langs = "en"
 # Default to Routes in the root package (and conf/routes)
 # !!!WARNING!!! DO NOT CHANGE THIS ROUTER
 play.http.router = prod.Routes
-
+play.http.errorHandler = "utils.ErrorHandler"
```

Comment:
There is already an error handler configured (line 40). Should probably move this line there and replace it.